runtime/src/balances.rs

```diff
@@ -438,8 +424,9 @@ decl_storage! {
 
 decl_module! {
     pub struct Module<T: Trait<I>, I: Instance = DefaultInstance> for enum Call where origin: T::Origin {
+        /// This is no longer needede but kept for compatibility reasons
         /// The minimum amount required to keep an account open.
-        const ExistentialDeposit: T::Balance = T::ExistentialDeposit::get();
+        const ExistentialDeposit: T::Balance = 0.into();
```

Comment:
This is set in runtime - why hard code it again here?

Reply:
We don't need to hard code it here. Hardcoding just saves an extra operation. Also, it removes the dependency of balances module on the `ExistentialDeposit` type. Albeit, I decided to not remove `ExistentialDeposit` type at the end because other modules needed it, I still removed the balances module's dependency on it.